data_hub_api/docmaps/sql/docmaps_index.sql

```diff
@@ -128,22 +128,31 @@ t_result AS (
     AND COALESCE(preprint_doi_and_url.preprint_doi, europepmc_response.doi) IS NOT NULL
 ),
 
-t_tdm_doi_and_path AS(
-  SELECT
-    DISTINCT
-    t_results.tdm_doi, 
-    t_results.tdm_path
-  FROM `elife-data-pipeline.prod.biorxiv_medrxiv_meca_path_metadata`
-  LEFT JOIN UNNEST(results) AS t_results
+t_latest_tdm_doi_and_path AS(
+  SELECT 
+  * EXCEPT(rn) 
+  FROM (
+    SELECT
+      ROW_NUMBER() OVER (
+        PARTITION BY  t_results.tdm_doi
+        ORDER BY imported_timestamp DESC
```

Comment:
Is it in this case maybe first the version (descending), and then imported_timestamp (descending)?
So that have the latest path to the latest version.
(Of course usually the latest imported data will be for a more recent version, but maybe not for our backfills that we didn't do all in sequential order)